### PR TITLE
Add tool_belt config for Katello 4.15

### PR DESCRIPTION
**Pull request by sambible**
Config for 4.15 release. Notably katello-host-tools and foreman_virt_who_configure are needing to be released, @chris1984 is working on virt_who, and I'm looking for release help for katello-host-tools within phoenix.

### CODE REVIEW

File: configs/katello/4.15.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    :branch: 1.15.z
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 4.5.0
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :katello-certs-tools:
+    :branch: 2.10.0  
+    :repo: https://github.com/Katello/katello-certs-tools.git
+  :foreman_virt_who_configure:
+    :branch: 0.5.24
```
Comment (chris1984) [inline configs/katello/4.15.yaml:20]: can you bump this to 0.5.25

File: configs/katello/4.15.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/3.13
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 1.15.z
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 4.5.0
```
Comment (chris1984) [inline configs/katello/4.15.yaml:14]: Let's bump this guy to 4.6.0 we can get a release out for it.

**Review by chris1984 — COMMENTED** [review]
A few small changes but looks good overall

**Review by chris1984 — APPROVED** [review]
Ack, looks good to me @ekohl can this get a review/merge?

**Comment by chris1984** [comment]
One thing I just remembered can you add the prior_releases and set it to 4.14.0 and 4.13.2

**Comment by sambible** [comment]
Yes, sorry - this was in a previous version of this document I had worked on, but got chopped when I was editing a different portion.

**Review by chris1984 — APPROVED** [review]

**Comment by ekohl** [comment]
> Ack, looks good to me @ekohl can this get a review/merge?

I thought I gave Katello release owners permissions to do so, but I only gave the team read permission (which is fairly redundant for an open source project). Updated that so you can now merge (as you can see by your green approval checkmark).

**Comment by ianballou** [comment]
Thanks all!

**Pull request merged**